spec/acceptance/api/internal/alert_limits_spec.rb

```diff
@@ -1,94 +1,100 @@
-resource 'Alert limits' do
-  header 'Accept', 'application/json'
-  header 'Content-Type', 'application/json'
+describe 'Alert limits' do
+  before do
+    header 'Accept', 'application/json'
+    header 'Content-Type', 'application/json'
+  end
 
-  get '/services/:service_id/alert_limits/' do
-    parameter :service_id, 'Service ID', required: true
-    let(:service_id) { '7575' }
+  context '/services/:service_id/alert_limits/' do
+    context 'GET' do
+      let(:service_id) { '7575' }
 
-    context 'when there are no alert limits' do
-      example 'Getting alert limits', document: false do
-        do_request
+      context 'when there are no alert limits' do
+        it 'Getting alert limits' do
+          get "/services/#{service_id}/alert_limits/"
 
-        expect(response_status).to eq(200)
-        expect(response_json['alert_limits']).to eq([])
+          expect(response_status).to eq(200)
+          expect(response_json['alert_limits']).to eq([])
+        end
       end
-    end
 
-    context 'when there are alert limits' do
-      before do
-        ThreeScale::Backend::AlertLimit.save(service_id, 50)
-        ThreeScale::Backend::AlertLimit.save(service_id, 100)
-      end
+      context 'when there are alert limits' do
+        before do
+          ThreeScale::Backend::AlertLimit.save(service_id, 50)
+          ThreeScale::Backend::AlertLimit.save(service_id, 100)
+        end
 
-      example_request 'Getting alert limits' do
-        expected_values = [{ "service_id" => service_id, "value" => 50 },
-                           { "service_id" => service_id, "value" => 100 }]
+        it 'Getting alert limits' do
+          expected_values = [{ "service_id" => service_id, "value" => 50 },
+                             { "service_id" => service_id, "value" => 100 }]
 
-        expect(response_status).to eq(200)
-        expect(response_json['alert_limits']).to eq(expected_values)
+          get "/services/#{service_id}/alert_limits/"
+
+          expect(response_status).to eq(200)
+          expect(response_json['alert_limits']).to eq(expected_values)
+        end
       end
     end
-  end
 
-  post '/services/:service_id/alert_limits/' do
-    parameter :service_id, 'Service ID', required: true
-    parameter :alert_limit, 'Limit value', required: true
+    context 'POST' do
+      let(:service_id)  { '7575' }
 
-    let(:service_id)  { '7575' }
-    let(:raw_post) { params.to_json }
```

Comment:
For reference:
https://github.com/zipmark/rspec_api_documentation/blob/d32bf17932a379052e8bb7540fc29ce7e72712ae/README.md#raw_post

Reply:
Thanks for the info 👍